src/components/titlePanel/createTitlePanel.ts

```diff
@@ -68,13 +79,30 @@ const createTitlePanel: TitlePanelFactory = createWidgetBase
 					};
 				}
 
+				titleProperties.role = 'heading';
+
+				if (ariaHeadingLevel) {
+					titleProperties['aria-level'] = String(ariaHeadingLevel);
+				}
+
+				if (open) {
+					// FIXME - id of content node
+					titleButtonProperties['aria-controls'] = 'content';
```

Comment:
The button should actually have `aria-controls="id string"` set all the time, both open and closed. (Incidentally, I believe Matt G recommended `uuid` to generate unique id's, if it helps)